### DISCUSSION TITLE
wildcards?

### DISCUSSION THREAD

**Discussion by megamanenm** [Acly/krita-ai-diffusion#1249] [Q&A]
Is there a syntax or something for the usage of wildcards? I.e., something like "_hair" which would randomly insert a color where the underscore is?

**Comment by Lunarbabe29** [comment]
We need a wildcard function, literally...

**Reply by Lunarbabe29** [reply]
I even have a syntax suggestion. Do one similar to NovelAI like idk... e.g. (||red hair|blue hair|green hair||) etc. Then the resulting image would have hair randomized between those options. It shouldn't be that hard, if I was good at code I would make one myself. But Please @Acly we could really use one.

**Reply by iyukar** [reply]
Wildcards would be great.

**Comment by nesmonmehdydeal** [comment]
Wildcard in Krita !!!

**Comment by Gushousekai195** [comment]
Wildcards in Krita... now

**Comment by iyukar** [comment]
A simple {lemon|apple|orange} option would already be great if it's easy to implement.
Something allowing for an identical wildcard to be used in multiple places of the prompt likely ideal for most.
What is the extend of wildcards people would use? A problem with the feature is that it's a wide spectrum from {lemon|apple|orange} to having to interpret links to contents of multiple text-files somewhere on the PC.

**Comment by Hamsterun** [comment]
As Wildcard are now available as a list, i wonder if it is possible to use files as Wildcards. Hope that will be possible one day.
